test/unit/reporters/html/html.test.js

```diff
@@ -32,6 +32,12 @@ describe('HTML Reporter', () => {
         });
     }
 
+    function emitResult_(options) {
```

Comment:
I did not move it, to the describe, where i use it, since it is used inside 2 describes under this one.

emitResult_ function updated, it now accepts result as argument